docs/node-configuration.md

```diff
@@ -237,6 +239,26 @@ where:
   [Unlock Wallet Configuration](#Unlock-Wallet-Configuration) section for
   structure details.
 
+### Consensus configuration
```

Comment:
s/configuration/Configuration to match the other headers style?

Reply:
I've specifically wanted to avoid that, but maybe consistency is a better choice.